### ISSUE TITLE
'__console__' instead of '__main__' as __name__ in bpython 0.20.1

### ISSUE THREAD

**Issue by colomb8** [bpython/bpython#868]
this works:
```
(py3) DC@bdd14:~/code/stable$ bpython
bpython version 0.20.1 on top of Python 3.8.5 /home/.../miniconda3/envs/py3/bin/python
>>> __name__
'__main__'
```

but this doesn't:
```
(py3) DC@bdd14:~/code/stable$ bpython -i main.py
bpython version 0.20.1 on top of Python 3.8.5 /home/.../miniconda3/envs/py3/bin/python
>>> __name__
'__console__'
```

**Comment by vmdhhh** [comment]
@sebastinas After a brief debugging, I found out that the `__name__` is getting changed to `__console__` in `__main__.py`.
And as we are importing the `main`, it gives `__console__`.
Sorry if this sounds obvious/rubbish as I am not a pro, but if you could direct me towards something, I would like to help.

**Comment by sebastinas** [comment]
The behavior should be consistent with the vanilla Python interpreter. Whatever it does, should be emulated here.

**Comment by vmdhhh** [comment]
> 
> 
> The behavior should be consistent with the vanilla Python interpreter. Whatever it does, should be emulated here.

@sebastinas  Does this mean that the `__name__` should output `__main__` everywhere?

**Comment by colomb8** [comment]
> > The behavior should be consistent with the vanilla Python interpreter. Whatever it does, should be emulated here.
> 
> @sebastinas Does this mean that the `__name__` should output `__main__` everywhere?

no, only in the main script, while in the others should return the name of each module

**Comment by sebastinas** [comment]
`python` has the following behavior:
```
% cat pm.py 
print(__name__)
% python3 pm.py
__main__
% python3 -i pm.py
__main__
>>> print(__name__)
__main__
% python3
>>> print(__name__)
__main__
```

`__name__` is set to `__console__` by  `code.InteractiveInterpreter`by default. I don't remeber if there was a specific reasons to keep this behavior. I think that we should align `bpython`'s behavior with that of `python`.

**Comment by sebastinas** [comment]
Fixed in 14669c0.
